# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xfree86/Imakefile.patch.gatos
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -dur gatos-ati.4.4.0-2004-08-09/Imakefile src.xfree86.1100824979.11936.2507168007/xc/gatos-ati.4.4.0-2004-08-09/Imakefile
--- gatos-ati.4.4.0-2004-08-09/Imakefile	2004-05-20 03:37:59.000000000 +0200
+++ src.xfree86.1100824979.11936.2507168007/xc/gatos-ati.4.4.0-2004-08-09/Imakefile	2004-11-19 15:29:46.355244728 +0100
@@ -239,18 +239,18 @@
 
 ModuleObjectRule()
 
-ObjectModuleTarget(ati,$(OBJS1))
-ObjectModuleTarget(atimisc,$(OBJS2))
-ObjectModuleTarget(r128,$(OBJS3))
-ObjectModuleTarget(radeon,$(OBJS4))
-ObjectModuleTarget(fi1236,$(OBJS_FI1236))
-ObjectModuleTarget(msp3430, $(OBJS_MSP3430))
-ObjectModuleTarget(theatre, $(OBJS_THEATRE))
-ObjectModuleTarget(bt829, $(OBJS_BT829))
-ObjectModuleTarget(saa7114, $(OBJS_SAA7114))
-ObjectModuleTarget(tda9850, $(OBJS_TDA9850))
-ObjectModuleTarget(tda8425, $(OBJS_TDA8425))
-ObjectModuleTarget(tda9885, $(OBJS_TDA9885))
+ObjectModuleTarget(ati,$(OBJS1),input)
+ObjectModuleTarget(atimisc,$(OBJS2),input)
+ObjectModuleTarget(r128,$(OBJS3),input)
+ObjectModuleTarget(radeon,$(OBJS4),input)
+ObjectModuleTarget(fi1236,$(OBJS_FI1236),input)
+ObjectModuleTarget(msp3430, $(OBJS_MSP3430),input)
+ObjectModuleTarget(theatre, $(OBJS_THEATRE),input)
+ObjectModuleTarget(bt829, $(OBJS_BT829),input)
+ObjectModuleTarget(saa7114, $(OBJS_SAA7114),input)
+ObjectModuleTarget(tda9850, $(OBJS_TDA9850),input)
+ObjectModuleTarget(tda8425, $(OBJS_TDA8425),input)
+ObjectModuleTarget(tda9885, $(OBJS_TDA9885),input)
 
 
 InstallObjectModule(ati,$(MODULEDIR),drivers)
